staking/parse_test.go

```diff
@@ -31,3 +34,73 @@ func BenchmarkParse(b *testing.B) {
 		require.NoError(b, err)
 	}
 }
+
+func TestValidateRSAPublicKeyIsWellFormed(t *testing.T) {
+	validKey, err := rsa.GenerateKey(rand.Reader, 2048)
+	require.NoError(t, err)
+
+	for _, testCase := range []struct {
+		description string
+		expectErr   error
+		getPK       func() rsa.PublicKey
+	}{
+		{
+			description: "valid public key",
+			getPK: func() rsa.PublicKey {
+				return validKey.PublicKey
+			},
+		},
+		{
+			description: "even modulus",
+			getPK: func() rsa.PublicKey {
+				evenN := new(big.Int).Set(validKey.N)
+				evenN.Add(evenN, big.NewInt(1))
+				return rsa.PublicKey{
+					N: evenN,
+					E: 65537,
+				}
+			},
+			expectErr: ErrRSAModulusIsEven,
```

Comment:
This is going to make my OCD really obvious lol
```suggestion
			expectErr: ErrRSAModulusIsEven,
			getPK: func() rsa.PublicKey {
				evenN := new(big.Int).Set(validKey.N)
				evenN.Add(evenN, big.NewInt(1))
				return rsa.PublicKey{
					N: evenN,
					E: 65537,
				}
			},
```

Reply:
I turn off mine when it comes to test code ;-)

Reply:
[Fixed](https://github.com/ava-labs/avalanchego/pull/3501/commits/e45a72ae90a2c993915320789a299e78557fccac#diff-f15b6d29bdd07ca1a1e4bb7a665f7c67eaf0ec5f2f6fae634c3619cc90efdc75R55)